File: dashboard/lib/dashboard/painting.ex
defmodule Dashboard.Painting do
  use Ecto.Schema
  import Ecto.Changeset

  schema "name" do
    field :artist, :string
    field :description, :string
    field :image_path, :string
    field :medium, :string
    field :year, :naive_datetime
    field :user_id, :id

    timestamps()
  end

  @doc false
  def changeset(painting, attrs) do
    painting
    |> cast(attrs, [:artist, :medium, :description, :image_path, :year])
    |> validate_required([:artist, :medium, :description, :image_path, :year])
  end
end

File: app/Pain/Pain/Model/Painting.swift
import UIKit

class Painting {
    var photo: UIImage?
    var title: String
    var description: String
    var artist: String
    var medium: String
    var museum: Int
    var year: String
    
    init(photo: UIImage?, title: String, description: String, artist: String, medium: String, museum: Int, year: String) {
        self.photo = photo
        self.title = title
        self.description = description
        self.artist = artist
        self.medium = medium
        self.museum = museum
        self.year = year
    }
}
